article/app/views/package.scala

```diff
@@ -72,7 +72,7 @@ object BodyCleaner {
       TimestampCleaner(article),
       MinuteCleaner(article),
       GarnettQuoteCleaner,
-      AffiliateLinksCleaner(request.uri, article.content.metadata.sectionId, article.content.fields.showAffiliateLinks, "article")
+      AffiliateLinksCleaner(request.uri, article.content.metadata.sectionId, article.content.fields.showAffiliateLinks, "article", tags = article.content.tags.tags.map(_.id))
```

Comment:
Would this be better taking `article` as an argument?

Reply:
it would be cleaner, it could also be a gallery so using these elements rather than the whole model makes things easier I think